### PR TITLE
build(dev-deps): update http-proxy-middleware from v2.0.6 to v3.0.0

### LINKED ISSUE #768: DeprecationWarning: 'onAfterSetupMiddleware' option is deprecated.
Jam was installed using the guides from [Raspibolt](https://raspibolt.org/guide/bonus/bitcoin/Jam.html) and still functions as expected since several months. However, there is a warning in the logs after starting jam after it was updated to  v0.2.0

## Steps to reproduce
  1. sudo systemctl start jam.service
  2. sudo journalctl -fu jam
  3. See Warning in logs, as shown below.

## System specifications
  - Version: v0.2.0 - Renewing Rhubarb
  - Operating system = ubuntu server
  - Browser: Firefox

```
ubuntu npm[75441]: [HPM] Proxy created: /obwatch/  -> http://127.0.0.1:62601
ubuntu npm[75441]: [HPM] Proxy rewrite rule created: "^/obwatch/" ~> ""
ubuntu npm[75441]: (node:75441) [DEP_WEBPACK_DEV_SERVER_ON_AFTER_SETUP_MIDDLEWARE] DeprecationWarning: 'onAfterSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
ubuntu npm[75441]: Starting the development server...
ubuntu npm[75441]: Compiled successfully!
ubuntu npm[75441]: You can now view jam in the browser.
```

### PR DESCRIPTION
**Pull request by theborakompanioni**
Resolves #768.

Updates dev dependency `http-proxy-middleware` from `v2.0.6` to `v3.0.0`.
See the migration guide for more information: https://github.com/chimurai/http-proxy-middleware/blob/master/MIGRATION.md

## How to test
- checkout branch
- run `npm install`
- run `npm run dev:start` and/or `npm run start`
- verify everything is running properly as before (e.g. create/import a wallet, attempt collaborative transaction, etc.)

### CODE REVIEW

**Comment by theborakompanioni** [comment]
> 🙏

:pray: 

I assume it works as expected? : -)
Just waiting for one more approval before merging. Thank you so much @httpiga

**Review by barrytra — APPROVED** [review]
Approved! It works as expected

**Pull request merged**
